daemon/src/maker_cfd_actor.rs

```diff
@@ -157,6 +167,8 @@ pub fn new(
                             .unwrap();
                     }
                     maker_cfd_actor::Command::StartContractSetup { taker_id, order_id } => {
+                        println!("CONTRACT SETUP");
```

Comment:
We need a logger.

Reply:
#94 (I converted the ToDo that was still there to an issue)